client/src/App.js

```diff
@@ -18,6 +70,34 @@ export default App;
 
 const Buffer = styled.div`
   height: 1px;
+  margin-bottom: 100px;
```

Comment:
rem might be more practical than px

Reply:
It is a placeholder, therefore it's called button. As of now, it needs a height for the sake of it being there. 1px is fine. Also what would 1px be in rem?